src/js/flags/model.ts

```diff
@@ -0,0 +1,23 @@
+export interface Preset {
```

Comment:
I find "model" a bit too ambiguous here, since it could easily be flags about the randomizer's model of the game, or something.  Can we instead name this "flag.ts"?

Reply:
I went ambiguous on the naming here and for `util`, since they're both grouped in the `flag` package. We can rename them however; I'm more a fan of just getting things broken out a bit for readability/maintainability.

Reply:
Thanks